### PR TITLE
website: Add BookInfo tutorial

### PR DESCRIPTION
**Pull request by kfirtoledo**
Add BookInfo tutorial with different load-balancing and privileged access policy examples.
https://deploy-preview-544--clusterlink-net.netlify.app/docs/main/tutorials/bookinfo/

### CODE REVIEW

File: website/content/en/docs/main/tutorials/bookinfo/index.md [hunk 1]
```diff
@@ -0,0 +1,403 @@
...
+                    clusterlink/metadata.gatewayName: server1
+                }
+    }
+" | kubectl apply -f -
+```
+
+{{% /tab %}}
+{{< /tabpane >}}
+
+To remove the privileged access policy use he following command:
```
Comment (zivnevo) [inline website/content/en/docs/main/tutorials/bookinfo/index.md:224]:
```suggestion
To remove the privileged access policy use the following command:
```

File: demos/bookinfo/kind/README.md
[the remark this answers is not in the document]
Comment (kfirtoledo) [reply demos/bookinfo/kind/README.md:1]: I need it to run the demo myself and not step by step...

[the remark this answers is not in the document]
Comment (kfirtoledo) [reply demos/bookinfo/kind/README.md:1]: Done

**Review by zivnevo — APPROVED** [review]

File: website/content/en/docs/main/tutorials/bookinfo/index.md [hunk 2]
```diff
@@ -0,0 +1,403 @@
+---
+title: BookInfo
+description: Running BookInfo application with different policies.
+---
+
+
+The tutorial sets up [Istio BookInfo application][Istio-BookInfo] in different clusters.
+The tutorial demonstrate the use of AccessPolicy and PrivilegedAccessPolicy custom resources.
```
Comment (michalmalka) [inline website/content/en/docs/main/tutorials/bookinfo/index.md:8]: demonstrates

File: website/content/en/docs/main/tutorials/bookinfo/index.md [hunk 3]
```diff
@@ -0,0 +1,403 @@
...
+The tutorial shows different load-balancing policies like: random, round-robin or static destination.
+For more details, see the [policies documentation][concept-policy].
+This test create three kind clusters:
```
Comment (michalmalka) [inline website/content/en/docs/main/tutorials/bookinfo/index.md:11]: creates

File: website/content/en/docs/main/tutorials/bookinfo/index.md [hunk 4]
```diff
@@ -0,0 +1,403 @@
...
+
+* Two Product-Page microservice (application frontend) and details microservice run on the first cluster.
+* The Reviews-V2 (display rating with black stars) and Rating microservices run on the second cluster.
+* The Reviews-V3 (display rating with black stars) and Rating microservices run on the third cluster.
```
Comment (michalmalka) [inline website/content/en/docs/main/tutorials/bookinfo/index.md:15]: red stars?

**Review by michalmalka — APPROVED** [review]

**Pull request merged**